optimize/apis-tools/optimize-api/optimize-api-authorization.md

```diff
@@ -21,7 +21,7 @@ Refer to [Public API Configuration](../../self-managed/optimize-deployment/confi
 
 You must obtain a token to use the Optimize API. When you create an Optimize [client](https://docs.camunda.io/docs/guides/setup-client-connection-credentials/), you get all the information needed to connect to Optimize.
 
-See our guide on [building your own client](https://docs.camunda.io/docs/apis-clients/build-your-own-client/).
+See our guide on [building your own client](https://docs.camunda.io/docs/apis-tools/build-your-own-client/).
```

Comment:
```suggestion
See our guide on [building your own client]($docs$/apis-tools/build-your-own-client/).
```

blocking: 

1. I don't think this URL should go to apis-tools, because it's a v8.1 URL. 
2. This should probably not be hardcoded to production anyway.